CHANGELOG.md

```diff
@@ -8,6 +8,23 @@ Try to keep listed changes to a concise bulleted list of simple explanations of
 
 ## [Unreleased]
 
+## [1.3.4] - 2023-10-17
+
+[Full changelog](https://github.com/openfga/openfga/compare/v1.3.3...v1.3.4)
+
+### Fixed
+
```

Comment:
Why extra newline here and below?

Reply:
🤔 so it looks like the previous release's changelog